x-pack/plugins/observability/server/assets/component_templates/slo_mappings_template.ts

```diff
@@ -50,34 +58,10 @@ export const getSLOMappingsTemplate = (name: string) => ({
             revision: {
               type: 'long',
             },
-            groupBy: {
```

Comment:
I think we can remove it from the mapping and the SLO transform pivot/runtime mapping _but_ we need to trigger a revision change when it changes, because the data needs to be recomputed.